packages/macos/generate_wazuh_packages.sh

```diff
@@ -123,16 +125,32 @@ function get_pkgproj_specs() {
 function build_package() {
 
     # Download source code
-    git clone --depth=1 -b ${BRANCH_TAG} ${WAZUH_SOURCE_REPOSITORY} "${WAZUH_PATH}"
-    short_commit_hash="$(curl -s https://api.github.com/repos/wazuh/wazuh/commits/${BRANCH_TAG} \
-                          | grep '"sha"' | head -n 1| cut -d '"' -f 4 | cut -c 1-7)"
+    if [ -n "$BRANCH_TAG" ]; then
+        SOURCES_DIRECTORY="${CURRENT_PATH}/repository"
+        WAZUH_PATH="${SOURCES_DIRECTORY}/wazuh"
+        git clone --depth=1 -b ${BRANCH_TAG} ${WAZUH_SOURCE_REPOSITORY} "${WAZUH_PATH}"
+        short_commit_hash="$(curl -s https://api.github.com/repos/wazuh/wazuh/commits/${WAZUH_BRANCH} \
+                | grep '"sha"' | head -n 1| cut -d '"' -f 4 | cut -c 1-11)"
```

Comment:
Could you check if this works:
```suggestion
short_commit_hash=$(git rev-parse -C "${WAZUH_PATH}" --short HEAD)
```

Reply:
done